setup.py

```diff
@@ -27,6 +27,7 @@
 
 kerberos_require = ["requests_kerberos"]
 sqlalchemy_require = ["sqlalchemy~=1.3"]
+localstorage_require = ["keyring"]
```

Comment:
add this to `all_require`. `all` must install everything (except test requirements).

Reply:
I'm not too sure about making this optional though.

Reply:
Discussed in https://github.com/trinodb/trino-python-client/pull/162#discussion_r841429442